parser.go

```diff
@@ -1061,10 +1063,17 @@ func (parser *Parser) visit(path string, f os.FileInfo, err error) error {
 }
 
 // Skip returns filepath.SkipDir error if match vendor and hidden folder
-func Skip(f os.FileInfo) error {
-	// exclude vendor folder
-	if f.IsDir() && f.Name() == "vendor" {
-		return filepath.SkipDir
+func (parser *Parser) Skip(path string, f os.FileInfo) error {
+
+	if parser.ParseVendor {
+		// Ignore this project
+		if f.IsDir() && strings.HasSuffix(path, "github.com/swaggo/swag") {
+			return filepath.SkipDir
+		}
+	} else { // ignore vendor
+		if f.IsDir() && f.Name() == "vendor" {
+			return filepath.SkipDir
+		}
```

Comment:
Is the code below has been used?

Reply:
Sry I my miss.

Reply:
but I think difficult code. I want to more simple code.
Do you have idea?

Reply:
I can change the comments in operation.go, to make it pass the build.

Reply:
OK. I good feel this approach